src/doc/src/SUMMARY.md

```diff
@@ -92,6 +92,10 @@
         * [cargo package](commands/cargo-package.md)
         * [cargo publish](commands/cargo-publish.md)
         * [cargo yank](commands/cargo-yank.md)
+    * [External Commands](commands/external-commands.md)
+        * [cargo clippy](commands/cargo-clippy.md)
+        * [cargo fmt](commands/cargo-fmt.md)
+        * [cargo miri](commands/cargo-miri.md)
```

Comment:
> Is “External commands” the right name to put in The Cargo Book?

The main question I have is whether its worth distinguishing these as external or to list them with the others.

Reply:
That also works, and might be better, though `cargo-fmt` is pretty hard to categorize. It doesn't involve either build or manidest

Reply:
Likely it should follow clippy

Reply:
They are all under build commands. If we find a better to categorize them we can do it later.